CHANGES.md

```diff
@@ -1,5 +1,89 @@
 # Changelog
 
+## 5.8 (2022-11-29)
```

Comment:
This date might have to be tweaked -- bit of an oopsie on my part for running this workflow so late at night.

Reply:
Fixed.